### PR TITLE
feat(dashboard): adding dashboard navigation and bar

### PR DESCRIPTION
**Pull request by SpiritGY**
# Description

Adding Navigation on Dashboard and NavigationBar

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to a clickup task
- [x] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

**Comment by Lucas-COX** [comment]
Task linked: [CU-860t03zvf Mise en place du Dashboard Monopage](https://app.clickup.com/t/860t03zvf)

**Review by lucas-louis — COMMENTED** [review]
It seems good but the github actions need to pass before approving this PR (I need to test the apk, only available if the GA pass)

⚠️ Use a better name for the PR

File: medecin_mobile/lib/models/dashboard.dart [hunk 1]
```diff
@@ -18,36 +20,38 @@ class _DashBoardState extends State<DashBoard> {
...
+        child:Padding (
+          padding: const EdgeInsets.symmetric(horizontal: 16.0, vertical: 8.0),
+          child: Stack(
+            children: [
+                      AnimatedSwitcher(
+                        duration: const Duration(milliseconds: 600),
+                        child: Padding(
+                          padding: const EdgeInsets.only(top: 80,),
+                            child : 
+                              pages[_selectedIndex],
```
Comment (MarvinFlm) [inline medecin_mobile/lib/models/dashboard.dart:50]: Fait attention au indentation la y a pas forcément besoin de le mette a la ligne

**Review by MarvinFlm — APPROVED** [review]
Good

**Review by lucas-louis — APPROVED** [review]
LGTM

**Review by MarvinFlm — APPROVED** [review]
Approve

**Pull request merged**